app/main.py

```diff
@@ -1,8 +1,50 @@
+import decimal
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) \
+            -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: int, clean_power: int,
+                 average_rating: decimal, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> decimal:
+        income_of_cars = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income_of_cars += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+            else:
+                continue
+        return round(income_of_cars, 1)
+    """method then return income of wash station and change clean_mark of car,
+    if clean_mark less than clean_power of wash station"""
+
+    def calculate_washing_price(self, car: type) -> decimal:
```

Comment:
The `type` keyword is used incorrectly here. You should specify the type of `car` as `Car` instead of `type`.